corona-warn-app/cwa-app-android
# Fix behavior of last 3 hours test mode switch (EXPOSUREAPP-3179)

Previously the days for which the last 3 hours were fetched, was the latest day in the servers index for a country, but if a day is not complete, the day will not be visible in the index. This PR fixes this behavior by artificially adding the one additional date (`LATEST_DAY_IN_INDEX+1DAY`).

## Testing
* Try it without this PR, check in the logs what the `KeyFileDownloader` downloads.
* Try with this PR, check that the actual last 3 hours are downloaded.
